### ISSUE TITLE
Feature request: add upload file path to log data

### ISSUE THREAD

**Issue by bh-teufels** [pagemachine/typo3-formlog#152]
the uploaded files get logged with file name, it would be awesome when also the path get logged.

on upload the files get saved in the selected storage inside a generated folder like "form_7c4bf264b5c1236827517ad756d0c71988d24296"

so it would be helpfull if i could get the storage path from the formlog entry

thanks

**Comment by mbrodala** [comment]
Good idea, can you post a PR? Adding this and adjusting the tests should be rather easy.

**Comment by bh-teufels** [comment]
it is indeed really simple just add 

`'path' => $formValue->getIdentifier(),` to LoggerFinisher.php -> getFormValues() at line 79

```
[...]
if ($formValue instanceof CoreFileReference) {
                    debug($formValue);
                    $normalizedFormValues[$identifier] = [
                        'file' => [
                            'name' => $formValue->getName(),
                            'path' => $formValue->getIdentifier(),
                        ],
                    ];
                    continue;
                }
[...]
```

**Comment by bh-teufels** [comment]
i also would add
`'url' => $formValue->getPublicUrl(),`
which links to the file so it is possible to directly view and download the file

**Comment by mbrodala** [comment]
Notice that the file identifier doesn't need to be its path. While this is the case for the `local` driver, this may be different for other storages using e.g. remote drivers. Instead one would need to use `$fileReference->getForLocalProcessing(false)`, but this may lead to a local copy of remote files in case of remote storages.

But `$fileReference->getPublicUrl()` should be safe to add.
